addons/rook/1.10.11/host-preflight.yaml

```
@@ -24,7 +24,7 @@ spec:
           when: "{{kurl if (and .Installer.Spec.Rook.Version .Installer.Spec.Rook.BlockDeviceFilter) }}{{kurl .Installer.Spec.Rook.BlockDeviceFilter }}{{kurl else }}.*{{kurl end }} > 1"
           message: Multiple available block devices
       - fail:
-          message: No available block devices
+          message: "No available block devices unformatted and dedicated to be use by Rook only were found attached to each node in the cluster. For further information see: https://kurl.sh/docs/add-ons/rook#block-storage"
```

>No available block devices unformatted and dedicated to be use by Rook only were found attached to each node in the cluster.

This is IMO not clear for a few reasons

1. "each node in the cluster" - this check only runs on the first node, and so this message is redundant
2. "dedicated to be use by Rook only" - this seems like it might be a source of confusion. How does one designate this? (they don't, but that's not clear)

I would instead say:
```suggestion
          message: "No available unformatted block devices were found, and Rook requires one. For further information see: https://kurl.sh/docs/add-ons/rook#block-storage"
```